requirements.in

```
@@ -2,7 +2,7 @@
 
 datacube[s3]==1.8.6
 aiobotocore[awscli, boto3]==2.1.0
```

You might need to change this to 2.2.0 to get the build working. And maybe to add `s3fs`... I can't work out why, but it wouldn't run for me without it in my local environment.

```suggestion
aiobotocore[awscli, boto3]==2.2.0
```

---

It succeeded, but I'll update that anyway. Thanks.